test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -177,7 +199,7 @@ BOOST_AUTO_TEST_CASE(double_stateVariable_declaration)
 					   "  uint256 variable;\n"
 					   "  uint128 variable;\n"
 					   "}\n";
-	BOOST_CHECK(expectError(text) == Error::Type::DeclarationError);
+	CHECK_ERROR(text, DeclarationError, "");
```

Comment:
One benefit I could see making these functions is that we could have two overloaded versions:

`check_error(text, DeclarationError);` vs `check_error(text, DeclarationError, "uint128 is invalid");`